backend/django_core/templates/user/register/_3.html

```diff
@@ -0,0 +1,52 @@
+<div class="flex flex-col gap-2 whitespace-nowrap font-bold uppercase leading-none tracking-widest text-white md:text-[1.2vw] md:gap-[0.5vw]">
+    <span class="flex items-center text-base font-bold uppercase tracking-widest text-white md:text-[1.2vw]">
+        welcome to&nbsp;
+        <p class="inline-flex">
+            <span class="text-white/90">c</span>
+            <span class="text-warning">o</span>
+            <span class="text-white/90">r</span>
+            <span class="text-white/90">e</span>
+        </p>
+        project
+    </span>
+    <span class="uppercase md:text-[1vw] font-medium text-white/90">Your Anime Sanctuary Awaits!</span>
+</div>
+<div class="flex flex-col items-start gap-[3vw]">
+    <user-info class="flex flex-col md:gap-[1.5vw]">
+        <span class="text-lg font-semibold text-primary-500 md:text-[1.5vw] leading-none">Account Summary</span>
+        <username class="flex flex-col md:gap-[0.5vw]">
+            <span class="text-lg font-medium md:text-[1.1vw] leading-none">Username:</span>
+            <span class="text-base font-medium text-accent md:text-[1.1vw] leading-none">{{ username }}</span>
+        </username>
+        <email class="flex flex-col md:gap-[0.5vw]">
+            <span class="text-lg font-medium md:text-[1.1vw] leading-none">Email:</span>
+            <span class="text-base font-medium text-accent md:text-[1.1vw] leading-none">{{ email }}</span>
+        </email>
+    </user-info>
+    <div class="mt-3 flex flex-col items-start md:mt-0">
+        <button class="btn !bg-secondary p-0 text-base font-semibold leading-none text-primary underline md:text-[1vw] flex flex-col items-start">
+            &lt; resend code &gt;
+        </button>
+        <button class="btn !bg-secondary p-0 text-base font-semibold leading-none text-primary underline md:text-[1vw] flex flex-col items-start">
+            &lt; change email &gt;
+        </button>
+    </div>
+</div>
+<div class="flex items-center justify-between">
+    <div class="flex flex-col gap-1 md:gap-[0.5vw]">
+        <span class="text-xs leading-none text-surface-100 md:text-[0.75vw]">Already have an account?</span>
+        <a
+            href="{% url 'login_view'%}"
+            class="text-base leading-none md:text-[1.1vw] text-primary underline"
+        >
+            Login
```

Comment:
Should this be login?

Reply:
Yes? no?
its register page so login? no? :confused: